src/test/java/TestUtils.kt

```diff
@@ -4,7 +4,7 @@ import java.nio.file.Paths
 
 object CodeSyncTestUtils {
     private val DIRECTORY_PATH = Paths.get(System.getProperty("user.dir"), "test_data").toAbsolutePath()
-    private val DATABASE_FILE = Paths.get(DIRECTORY_PATH.toString(), "file.db").toAbsolutePath()
+    private val DATABASE_FILE = Paths.get(DIRECTORY_PATH.toString(), "test.db").toAbsolutePath()
```

Comment:
Kindly add a TODO here that this should come from config (relative to root repo). We should not define it here.